swarm/bmt/bmt.go

```diff
@@ -377,7 +408,8 @@ func (h *Hasher) Write(b []byte) (int, error) {
 // it reads so that chunk to hash is maximum length or reader reaches EOF
 // caller must Reset the hasher prior to call
 func (h *Hasher) ReadFrom(r io.Reader) (m int64, err error) {
```

Comment:
I don't see this implementation of `io.ReaderFrom` used anywhere or tested?

Reply:
indeed, i add test or remove and TODO